src/ssl.c

```diff
@@ -1013,6 +1013,22 @@ int wolfSSL_mutual_auth(WOLFSSL* ssl, int req)
 }
 #endif /* NO_CERTS */
 
+#ifdef WOLFSSL_WOLFSENTRY_HOOKS
+
+WOLFSSL_API int wolfSSL_CTX_set_AcceptFilter(WOLFSSL_CTX *ctx, NetworkFilterCallback_t AcceptFilter, void *AcceptFilter_arg) {
```

Comment:
Only put `WOLFSSL_API` on the header. This is a public API please add check for `ctx != NULL`. Same for `wolfSSL_set_AcceptFilter`

Reply:
fixed.